master/static/migrations/20230503200905_backfill_tasks.tx.up.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE tasks ADD COLUMN trial_id INTEGER NULL;
+
+INSERT INTO tasks (task_id, task_type, start_time, end_time, job_id, log_version, trial_id)
+SELECT
+    'backfilled-' || t.id,
```

Comment:
```suggestion
    t.experiment_id || '.backfilled-' || t.id,
```
otherwise `experimentIDFromTrialTaskID` breaks
https://github.com/determined-ai/determined/pull/6519/files#diff-4560e8503ef73d7b056675ba16ec4cc13255101112f421ba2826790a81d427d6R2205

Reply:
great catch, glad you have done this already